lib/menu.rb

```diff
@@ -0,0 +1,11 @@
+require_relative './takeaway.rb'
+
+class Menu
+
+  attr_reader :items
+
+  def initialize
+    @items = [{:pizza=>"£9"}, {:pasta=>"£7"}]
+  end
+
```

Comment:
Could have a method that prints it out and then gets rid of havigng pound and string in the hash.